### PR TITLE
support to install by olm

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -82,6 +82,12 @@ verify-scripts:
 .PHONY: verify-scripts
 verify: verify-scripts verify-codegen-crds
 
+deploy-operator: ensure-operator-sdk
```
Comment (skeeey) [inline Makefile:85]: could we change this task name? e.g. deploy-with-olm or else, this name may not be straightforward

Comment (zhiweiyin318) [reply Makefile]: done

**Pull request merged**